packages/fluentui/react-bindings/src/hooks/useOnFrameFocus.ts

```diff
@@ -0,0 +1,10 @@
+import { useFrameFocusDispatch } from './useFrameFocusDispatch';
+import { useFrameListener } from './useFrameListener';
+
+/**
+ * It enabables a interval to check if the iframe is focused and executes the callback function.
+ */
+export const useOnFrameFocus = (enableFrameFocusDispatch: boolean, $document: Document, cb: (e: Event) => void) => {
+  useFrameFocusDispatch(enableFrameFocusDispatch, $document);
+  useFrameListener(enableFrameFocusDispatch, cb, $document);
+};
```

Comment:
As we handle events only for `iframes` I suggest to update naming to include "i", i.e. "useOnIFrameFocus" and etc. As "frame" is a different element, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frame.